digits/dataset/images/classification/test_views.py

```diff
@@ -66,7 +66,7 @@ class BaseViewsTestWithImageset(BaseViewsTest):
     IMAGE_WIDTH     = 10
     IMAGE_CHANNELS  = 3
     BACKEND         = 'lmdb'
-    COMPRESSION     = 'none'
+    COMPRESSION     = 'none' 
```

Comment:
Ok I finally installed a sublime plugin to highlight those trailing spaces! I should be able not to add any new trailing space at least.